Build failure using version 2.2.0 - missing 'broccoli-file-creator' dependency

In 2.2.0, require('broccoli-file-creator') was added to /index.js but not to package.json. Using this version in my application produces the following error:
```
Cannot find module 'broccoli-file-creator'
```
When I force the version back to 2.1.1 in my package.json, the error no longer occurs.  (I debated submitting the issue or making a PR, but things are hectic this evening and I didn't want this to slip through the cracks.)

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
     "broccoli-merge-trees": "^3.0.1",
     "broccoli-plugin": "^1.3.1",
     "chalk": "^2.4.1",
+    "cheerio": "^1.0.0-rc.3",
     "ember-cli-babel": "^7.1.0",
     "ember-cli-lodash-subset": "2.0.1",
     "ember-cli-preprocess-registry": "^3.1.2",
@@ -41,11 +42,11 @@
   "devDependencies": {
     "body-parser": "^1.18.3",
     "broccoli-asset-rev": "^3.0.0",
+    "broccoli-file-creator": "^1.1.1",
     "broccoli-test-helper": "^1.5.0",
     "chai": "^4.1.2",
     "chai-fs": "^2.0.0",
     "chai-string": "^1.4.0",
-    "cheerio": "^1.0.0-rc.3",
     "co": "^4.6.0",
     "ember-ajax": "^3.1.0",
     "ember-cli": "~3.3.0",
@@ -75,8 +76,7 @@
     "release-it": "^12.0.1",
     "release-it-lerna-changelog": "^1.0.2",
     "request": "^2.88.0",
-    "rsvp": "^4.8.3",
-    "broccoli-file-creator": "^1.1.1"
+    "rsvp": "^4.8.3"
   },
   "engines": {
     "node": "6.* || 8.* || >= 10.*"

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "broccoli-concat": "^3.7.1",
+    "broccoli-file-creator": "^2.1.1",
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^3.0.1",
     "broccoli-plugin": "^1.3.1",
@@ -42,7 +43,6 @@
   "devDependencies": {
     "body-parser": "^1.18.3",
     "broccoli-asset-rev": "^3.0.0",
-    "broccoli-file-creator": "^1.1.1",
     "broccoli-test-helper": "^1.5.0",
     "chai": "^4.1.2",
     "chai-fs": "^2.0.0",

